suspendmanager.lua

```diff
@@ -378,6 +390,20 @@ function SuspendManager:shouldStaySuspended(job)
     return self.suspensions[job.id]
 end
 
+--- Return a human readable description of why suspendmanager keeps a job suspended
+--- or nil if the job is not kept suspended
+function SuspendManager:suspensionDescription(job)
```

Comment:
It might be more consistent to always have a message on suspended buildings. If a player clicks on a suspended building to discover the reason, and nothing is displayed, it might be confusing. Instead of returning nil, perhaps a message like "Unknown (not suspended by DFHack suspendmanager)"?

Reply:
Agreed, I went for the more concise "Suspended because: External interruption.". This feels vague and right enough I think?